src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCosts = 40;
+  const totalCost = dayCosts * days;
+  const baseDiscount = 20;
+  const wideDiscount = 50;
```

Comment:
In general, the term "wide discount" is not a commonly used term in the context of discounts or promotion, so it might not be clear, change it for "large" "major" or something like that